### PR TITLE
change icons in confirm dialogs boxes

### PR DESCRIPTION
**Pull request by ct-goranpalibrk**
Signed-off-by: CT\pgoran <goran.palibrk@comtrade.com>

Brief description of the PR.
Changed icons in confirmation dialogs.

**Related Issue**
This PR fixes issue #2338 

**Description of the solution adopted**
Changed css class for information icons in confirm dialogs.

**Screenshots**
If applicable, add screenshots to help explain your solution

**Any side note on the changes made**
Description of any other change that has been made, which is not directly linked to the issue resolution
[e.g. Code clean up/Sonar issue resolution]

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR successfully resolves issue #2338.

**Comment by ct-goranpalibrk** [comment]
@lorthirk Is this a good solution, or do you have another proposal?.
Thanks,
Goran.

**Comment by muros-ct** [comment]
If @lorthirk is ok with this I can merge. It is not solution but a workaround to solve the issue.

**Comment by LeoNerdoG** [comment]
I have verified that new changes are OK from console point of view 😄

**Comment by ct-goranpalibrk** [comment]
Hi @lorthirk , @muros-ct I have adopted another solution. Please take a look.

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import com.extjs.gxt.ui.client.widget.MessageBox;
+
+public class KapuaMessageBox extends MessageBox{
+
+    public static String question = "fa fa-question-circle fa-3x";
+    private Dialog dialog;
+    public static final String YESNO = Dialog.YESNO;
+
+    public static MessageBox confirm(String title, String msg, Listener<MessageBoxEvent> callback) { 
+          KapuaMessageBox box = new KapuaMessageBox();
```
Comment (lorthirk) [inline console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java]: Have you tried removing completely this method, and then do both a `setIcon(question)` and a `setIconStyle(question)` inside `getDialog()`? I suspect you may achieve the same behaviour without this method.

Comment (ct-goranpalibrk) [reply console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java]: I have tried that but it doesn't work. :)

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+package org.eclipse.kapua.app.console.module.api.client.ui.dialog;
+
+import com.extjs.gxt.ui.client.event.Listener;
+import com.extjs.gxt.ui.client.event.MessageBoxEvent;
+import com.extjs.gxt.ui.client.widget.Dialog;
+import com.extjs.gxt.ui.client.widget.MessageBox;
+
+public class KapuaMessageBox extends MessageBox{
+
+    public static String question = "fa fa-question-circle fa-3x";
```
[the remark this answers is not in the document]
Comment (ct-goranpalibrk) [reply console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java]: Question variable is now final.

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      }
+
+    @Override
+    public Dialog getDialog() {
+        dialog = super.getDialog();
+        dialog.setIconStyle(question);
+        return dialog;
+    }
+
+    @Override
```
[the remark this answers is not in the document]
Comment (ct-goranpalibrk) [reply console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java]: Override annotation removed.

[the remark this answers is not in the document]
Comment (lorthirk) [reply console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/dialog/KapuaMessageBox.java]: I meant the whole methode :)

**Comment by LeoNerdoG** [comment]
i have verified that everything is OK after last commit. 
@lorthirk can you please check the code if it is ok now?

**Comment by lorthirk** [comment]
LGTM.

**Comment by lorthirk** [comment]
Oh, I see that the Eclipse verification is failing. Please wait before merging.

**Pull request merged**